### ISSUE TITLE
Add option to limit concurrent (duplicated) execution

### ISSUE THREAD

**Issue by umputun** [umputun/cronn#3]
In many cases running the same job as the previous job still running is something unexpected and should be avoided. The easiest solution is to add some `--no-dup` parameter and postpone or even reject scheduled job in such cases.

_I'm not sure if such a simple limiter will be granular enought and can imagine jobs we want to run in parallel and some we don't._
